### PR TITLE
Dilpreet | Test | Small design issues in Convert customer/vendor/aging-report in material

### PR DESCRIPTION
**Pull request by dilpreet77**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
https://app.clickup.com/t/1wn5e3j

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/contact/contact.component.scss [hunk 1]
```diff
@@ -385,25 +387,32 @@ span.input-addon {
 }
 
 button.btn.btn-filter {
-    background: $whisper-color;
-    border: 1px solid $whisper-color;
+    background: #eaeaea;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/contact/contact.component.scss:390]: Avoid using hard coded color values; do use color variables

File: apps/web-giddh/src/app/contact/contact.component.ts [hunk 2]
```diff
@@ -1700,8 +1701,14 @@ export class ContactComponent implements OnInit, OnDestroy {
     public tabChange(event: any): void {
         if (event?.tab?.textLabel === this.localeData?.customer) {
             this.tabSelected("customer");
+            this.availableColumnsCount = [];
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/contact/contact.component.ts:1704]: Consider moving this code to a function as it is redundant at line 1704, 1709

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/contact/contact.component.ts [hunk 3]
```diff
@@ -1701,16 +1701,22 @@ export class ContactComponent implements OnInit, OnDestroy {
...
-            this.availableColumnsCount = [];
-            this.showFieldFilter = {};
-            this.translationComplete(true);
         } else if (event?.tab?.textLabel === this.localeData?.aging_report) {
             this.tabSelected("aging-report");
         }
     }
+
+    /**
+     *This function will use for reset columns
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/contact/contact.component.ts:1713]: Do leave single space before writing JS doc; add a single character space before `This`

File: apps/web-giddh/src/app/contact/contact.component.ts [hunk 4]
```diff
@@ -1701,16 +1701,22 @@ export class ContactComponent implements OnInit, OnDestroy {
     public tabChange(event: any): void {
         if (event?.tab?.textLabel === this.localeData?.customer) {
             this.tabSelected("customer");
-            this.availableColumnsCount = [];
-            this.showFieldFilter = {};
-            this.translationComplete(true);
+            this.resetColumns();
         } else if (event?.tab?.textLabel === this.localeData?.vendor) {
             this.tabSelected("vendor");
-            this.availableColumnsCount = [];
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/contact/contact.component.ts]: `resetColumns` is not called here, please check

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/contact/contact.component.ts (hunk 4 above)
Comment (dilpreet77) [reply apps/web-giddh/src/app/contact/contact.component.ts]: As per discussion , this function will not called here .

**Review by prateek-giddh — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**